packages/dds/tree/src/simple-tree/api/treeApiBeta.ts

```diff
@@ -91,11 +103,61 @@ export const TreeBeta = {
 	 * @returns A callback function which will deregister the event.
 	 * This callback should be called only once.
 	 */
+	on<K extends keyof TreeChangeEventsBeta<TNode>, TNode extends TreeNode>(
+		node: TNode,
+		eventName: K,
+		listener: NoInfer<TreeChangeEventsBeta<TNode>[K]>,
+	): () => void;
+
+	/**
+	 * Clones the persisted data associated with a node. Some key things to note:
+	 * - Local state, such as properties added to customized schema classes, will not be cloned. However, they will be
+	 * initialized to the state after running the constructor, just like if a remote client had inserted the same node.
+	 * - Primitive node types (i.e., numbers, strings, booleans, nulls and fluid handles) will be returned as is.
+	 * - The identifiers in the node's subtree will be preserved, i.e., they are not replaced with new values.
+	 *
+	 * @param node - The node to clone.
+	 * @returns A new unhydrated node with the same persisted data as the original node.
+	 */
+	clone<TSchema extends ImplicitFieldSchema>(
+		node: TreeFieldFromImplicitField<TSchema>,
+	): TreeFieldFromImplicitField<TSchema>;
+} = {
 	on<K extends keyof TreeChangeEventsBeta<TNode>, TNode extends TreeNode>(
 		node: TNode,
 		eventName: K,
 		listener: NoInfer<TreeChangeEventsBeta<TNode>[K]>,
 	): () => void {
 		return treeNodeApi.on(node, eventName, listener);
 	},
-} as const;
+	clone<TSchema extends ImplicitFieldSchema>(
+		node: TreeFieldFromImplicitField<TSchema>,
+	): Unhydrated<TreeFieldFromImplicitField<TSchema>> {
+		// If it's a primitive type (i.e., not a tree node), return it as is.
```

Comment:
While this code is correct, I think the comment is a bit too specific, since this handles undefined and fluid handles, neither of which are often thought of as primitives.

```suggestion
		// The only non TreeNode cases are TreeLeafValue and undefined (for an empty optional field) which can be returned as is:
```

Reply:
You could additionally (or instead) link to the `Value` type that is defined in types.ts - since that is exactly what we're trying to accept here. You might also want to assert `isTreeValue()`, just for sanity and robustness, even though I don't think it's strictly necessary if the input type is correct.